MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/Models/ListItem.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Web;
+
+namespace MyPerfectOnboarding.Api.Models
+{
+    [DataContract]
+    public class ListItem
```

Comment:
Your model should be just data transfer model, you do not need constructor and any private fields. All you need is plain object with properties - after you remove constructor, even displaying fields in browser will work.